R/calc_views.R

```diff
@@ -38,8 +38,9 @@ calc_view_lims <- function(views){
     view <- views[[i]]
     x <- lapply(view, var='x', function(list, var) strip_pts(list,var))
     y <- lapply(view, var='y', function(list, var) strip_pts(list,var))
-
-    views[[i]]$usr <- c(usr_from_lim(lims_from_list(x)), usr_from_lim(lims_from_list(y)))
+    xlim <- lims_from_list(x)
+    ylim <- lims_from_list(y)
+    views[[i]]$usr <- c(usr_from_lim(xlim, type=par()$xaxs), usr_from_lim(ylim, type=par()$yaxs))
```

Comment:
can't get this from par, since it may not be set by the call for this view. Need to get it from the view's `gs.config`